### PR TITLE
feat: add plane-node for v5 

### PR DESCRIPTION
**Pull request by GemT27**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Description of change
feat: 新增plane平面节点
feat: 添加demo案例

[screenity.webm](https://github.com/antvis/G6/assets/39400058/b7bffa50-0baf-4adf-8817-56ec4473095e)

### CODE REVIEW

**Comment by Aarebecca** [comment]
有变更的 API 麻烦更新下对应的文档呢，位于 `packages/site/docs/apis/shape/PlaneGeometryProps.zh/en.md` 下

File: packages/g6/src/util/shape3d.ts [hunk 1]
```diff
@@ -104,7 +106,7 @@ export const createShape3D = (
       ]);
       break;
     case 'plane':
-      shape.scale([style.width / GEOMETRY_SIZE, style.depth / GEOMETRY_SIZE]);
+      shape.scale(style.width / GEOMETRY_SIZE, style.depth / GEOMETRY_SIZE);
```
Comment (Aarebecca) [inline packages/g6/src/util/shape3d.ts:109]: `scale(scaling: vec3 | vec2 | number, y?: number, z?: number): this;` scale 第二个参数是 y 值，确认下是否正确呢

Comment (GemT27) [reply packages/g6/src/util/shape3d.ts]: 改好啦

Comment (Aarebecca) [reply packages/g6/src/util/shape3d.ts]: > 改好啦

如果有参加 IssueHunt 计划，请将相关的 Issue 回复到下面哦～

**Comment by GemT27** [comment]
#5094 添加三层拓扑-demo案例

File: packages/g6/src/util/shape3d.ts (hunk 1 above)
Comment (GemT27) [reply packages/g6/src/util/shape3d.ts]: 👌🏻

**Pull request merged**